### ISSUE TITLE
Images don't show up in IE8 with 1x/2x syntax

### ISSUE THREAD

**Issue by oilvier** [scottjehl/picturefill#239]
Hi,

I encountered a problem in Internet Explorer 8 (and below).

When using the syntax to display retina images, no images appear in the browser.

```
<img srcset="http://placehold.it/300x150 1x, http://placehold.it/600x300 2x" alt="Retina" />
```

I don't see any errors - the code adding the src to the img tag just doesn't seem to be applied.

Here is a fiddle with the test code : http://jsfiddle.net/oilvier/VdbT4/
(I used the code from the demo site to compare with a case where it's working)

If you watch it in IE8, you'll need this link, as jsfiddle interface is not displaying well in IE8 : http://fiddle.jshell.net/oilvier/VdbT4/show/light/

**Comment by jochienabuurs** [comment]
I still get this issue, using version 2.3.1. It seems that retina only image tags must be placed within a picture element.

This doesn't work:
```
<img srcset="http://placehold.it/300x150 1x, http://placehold.it/600x300 2x" alt="Retina" />
```
while this does:
```
<picture>
    <img srcset="http://placehold.it/300x150 1x, http://placehold.it/600x300 2x" alt="Retina" />
</picture>
```

**Comment by mike-engel** [comment]
@jochienabuurs Is this happening only on IE8?

**Comment by jochienabuurs** [comment]
@mike-engel I can't verify that now but will do that tomorrow. I've tested the code posted earlier in chrome, Safari (desktop and iOS 8) and IE11 and those all work correct. Will test tomorrow in FF, Opera and IE 9...

**Comment by mike-engel** [comment]
quick ping to @jochienabuurs to see if you've had time to test this yet.

**Comment by jochienabuurs** [comment]
I've tested the 2.3.1 version as well as an older version (don't know the number).
I can verify that the 2.3.1 version which i've pulled from the dist just a moment ago does not work in IE8.

I've set up a simple test environment:
https://github.com/jochienabuurs/polyfill-test

With the 2.3.1 version, i get the following results:
IE8 -> image tag without wrapped picture tag shows only alt text;
IE9 -> works as intended;
IE11 -> works as intended;
FF -> works as intended;
Chrome -> works as intended;
Opera -> works as intended.

I've also tested with an older version, with which i get the following results:
IE8 -> art direction doesn't work, but image tag without wrapped picture tag does;
IE9 -> art direction doesn't work, but image tag without wrapped picture tag does;
IE11 -> works as intended;
FF -> works as intended;
Chrome -> works as intended;
Opera -> works as intended.

The only thing missing from the new version is art direction for IE8 and IE9.
Perhaps comparing the two versions I supplied in my test would help u figure out what's wrong?

**Comment by aFarkas** [comment]
I had no time to look deeper into this. It seems, that pf2.x doesn't recognize that there is a ``srcset`` (i.e.: ``img[pf.ns].srcset`` is undefined.

However, I can confirm that this bug is fixed in version 3.0:
https://rawgit.com/scottjehl/picturefill/afarkas-pf3-proposal/dist/picturefill.js

**Comment by mike-engel** [comment]
@aFarkas Thoughts on backporting that to 2.x?

**Comment by jegtnes** [comment]
Seems like this one's been fixed in 2.3. @oilvier, if that's not the case, please re-open this.
